tests/wp-unit/include/Core/Metas/Modules/DeleteUserMetaModuleTest.php

```diff
@@ -29,18 +32,20 @@ public function setUp() {
 	/**
 	 * Add to test single user meta from admin user role.
 	 */
-	public function test_deleting_single_user_meta_fields_from_admin_user_role() {
+	public function test_that_delete_single_user_meta_fields_from_admin_user_role() {
```

Comment:
This naming is wrong.

It should be `test_that_meta_fields_from_a_single_user_in_admin_role_can_be_delete`

Change it for other methods.

Reply:
Fixed